apps/web/screens/Grants/SearchResults/SearchResults.tsx

```
@@ -0,0 +1,465 @@
+                        tagVariant = 'rose'
+                        break
+                      case GrantStatus.OpensSoon:
+                        tagVariant = 'purple'
+                        break
+                      default:
+                        break
+                    }
+
+                    return (
+                      <Box>
+                        {grant.applicationId && (
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)